.github/workflows/main.yml

```diff
@@ -28,7 +28,10 @@ jobs:
       run: |
         python --version
         pip --version
-        pip install --upgrade setuptools wheel
+        # XXX With setuptools 63.4.3 .. 64.0.0 the scintilla nmake won't compile (
```

Comment:
Seems related to #1897 and https://github.com/pypa/distutils/pull/153?

Reply:
Yes, problem reported over there. (Temporary) workaround and setuptools version unfreeze in #1936